editor/editor_help.cpp

```diff
@@ -1410,43 +1409,40 @@ Error EditorHelp::_goto_desc(const String &p_class, int p_vscr) {
 		}
 	}
 
-	if (method_descr) {
-
-		section_line.push_back(Pair<String, int>(TTR("Methods"), class_desc->get_line_count() - 2));
-		class_desc->push_color(title_color);
-		class_desc->push_font(doc_title_font);
-		class_desc->add_text(TTR("Method Description:"));
-		class_desc->pop();
-		class_desc->pop();
-
-		class_desc->add_newline();
-		class_desc->add_newline();
+	section_line.push_back(Pair<String, int>(TTR("Methods"), class_desc->get_line_count() - 2));
+	class_desc->push_color(title_color);
+	class_desc->push_font(doc_title_font);
+	class_desc->add_text(TTR("Method Description:"));
+	class_desc->pop();
+	class_desc->pop();
 
-		for (int i = 0; i < methods.size(); i++) {
+	class_desc->add_newline();
+	class_desc->add_newline();
 
-			_add_method(methods[i], false);
+	for (int i = 0; i < methods.size(); i++) {
 
-			class_desc->add_newline();
-			class_desc->push_color(text_color);
-			class_desc->push_font(doc_font);
-			class_desc->push_indent(1);
-			if (methods[i].description.strip_edges() != String()) {
-				_add_text(methods[i].description);
-			} else {
-				class_desc->add_image(get_icon("Error", "EditorIcons"));
-				class_desc->add_text(" ");
-				class_desc->push_color(comment_color);
-				class_desc->append_bbcode(TTR("There is currently no description for this method. Please help us by [color=$color][url=$url]contributing one[/url][/color]!").replace("$url", CONTRIBUTE_URL).replace("$color", link_color_text));
-				class_desc->pop();
-			}
+		_add_method(methods[i], false);
 
+		class_desc->add_newline();
+		class_desc->push_color(text_color);
+		class_desc->push_font(doc_font);
+		class_desc->push_indent(1);
+		if (methods[i].description.strip_edges() != String()) {
```

Comment:
For the referance, `method_descr` used to be set here to true in the past, but since we now have a `else` case to show a link to contributor's documentation, it's no longer needed (methods are always described).

Reply:
@akien-mga ops, this method is needed yet, can you revert this commit for me? I'll open another pr.

Reply:
@akien-mga or I can just open another pr without revert?

Reply:
New fix #14814